english/bamboo/bamboo-shoots.md

```diff
@@ -1,2 +1,4 @@
 at the top of the bamboo trees you meet a whole panda family, they are friendly.
-they show you a path ahead out of the forest, which you take.
\ No newline at end of file
+they show you a path ahead out of the forest, which you take.
+
+After five hours of treking you reach a waterfall. You decide to camp here for the night and start gathering branches -not [git ones](https://git-scm.com/docs/git-branch)- to light a fire.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.